src/scripts/OutSystems/OSUI/Patterns/OverflowMenuAPI.ts

```diff
@@ -0,0 +1,181 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace OutSystems.OSUI.Patterns.OverflowMenuAPI {
+	const _overflowMenuMap = new Map<string, OSFramework.OSUI.Patterns.OverflowMenu.IOverflowMenu>();
+
+	/**
+	 * Function that will change the property of a given OverflowMenu pattern.
+	 *
+	 * @export
+	 * @param {string} overflowMenuId
+	 * @param {string} propertyName
+	 * @param {*} propertyValue
+	 * @return {*}  {string}
+	 */
+	export function ChangeProperty(overflowMenuId: string, propertyName: string, propertyValue: unknown): string {
+		const result = OutSystems.OSUI.Utils.CreateApiResponse({
+			errorCode: ErrorCodes.OverflowMenu.FailChangeProperty,
+			callback: () => {
+				const overflowMenu = GetOverflowMenuById(overflowMenuId);
+
+				overflowMenu.changeProperty(propertyName, propertyValue);
+			},
+		});
+
+		return result;
+	}
+
+	/**
+	 * Create the new OverflowMenu instance and add it to the OverflowMenu Map
+	 *
+	 * @export
+	 * @param {string} overflowMenuId
+	 * @param {string} configs
+	 * @return {*}  {OSFramework.OSUI.Patterns.OverflowMenu.IOverflowMenu}
+	 */
+	export function Create(
+		overflowMenuId: string,
+		configs: string
+	): OSFramework.OSUI.Patterns.OverflowMenu.IOverflowMenu {
+		if (_overflowMenuMap.has(overflowMenuId)) {
+			throw new Error(
+				`There is already a ${OSFramework.OSUI.GlobalEnum.PatternName.OverflowMenu} registered under id: ${overflowMenuId}`
```

Comment:
@BenOsodrac Shouldn't we have an error code here?

Reply:
We usually don't have one for this momento. I followed the same approach as in other patterns API